parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueReader.java

```diff
@@ -30,4 +30,6 @@
   List<TripleIterator<?>> columns();
 
   void setPageSource(PageReadStore pageStore);
+
+  default void setRowOffsetForRowGroup(long position) {}
```

Comment:
Why not add the position to the page source? Then the two operations are tied together: the row offset is the start offset for the new pages.

Reply:
I can change to that. Just one thing that do we mind to change the function signature in the public API?